tdrs-backend/tdpservice/parsers/validators.py

```diff
@@ -439,7 +439,7 @@ def isSmallerThanOrEqualTo(UpperBound):
 def isInLimits(LowerBound, UpperBound):
     """Validate that value is in a range including the limits."""
     return make_validator(
-        lambda value: value >= LowerBound and value <= UpperBound,
+        lambda value: int(value) >= LowerBound and int(value) <= UpperBound,
```

Comment:
I don't think we should make a conversion here unless we are planning to get rid of `isInStringRange`. Im open to either option.